yoticlient.go

```diff
@@ -159,42 +160,67 @@ func handleHTTPError(response *httpResponse, errorMessages ...map[int]string) er
 			return fmt.Errorf(
 				defaultMessage,
 				response.StatusCode,
-				response.Content,
+				body,
 			)
 		}
 
 	}
 	return fmt.Errorf(
 		defaultUnknownErrorMessageConst,
 		response.StatusCode,
-		response.Content,
+		body,
 	)
 }
 
+func (client *Client) getDefaultHeaders() (headers map[string][]string) {
+	headers = map[string][]string{
+		sdkIdentifierHeader:        {sdkIdentifier},
+		sdkVersionIdentifierHeader: {sdkIdentifier + "-" + sdkVersionIdentifier},
+	}
+	return
+}
+
+// MakeRequest is used by other yoti Packages to make requests using a single
+// common client object. Users should not use this function directly
```

Comment:
If it's not exported, they can't use it anyway right?

Reply:
MakeRequest is exported.

Reply:
Comment must have snuck in a rebase or something.